apps/fz_wall/lib/fz_wall/cli/live.ex

```diff
@@ -161,7 +161,7 @@ defmodule FzWall.CLI.Live do
       get_filter_set_name(rule.user_id, ip_type, rule.action, layer4),
       rule.destination,
       port_type,
-      rule.port_range
+      FzHttp.Types.Int4Range.cast(rule.port_range)
```

Comment:
I'm not sure why at this point the `port_range` isn't an `Int4Range` already

Reply:
I think it may be because the rule hasn't been saved and loaded back from the DB yet, so it's still a `Postgrex.Range` type and prints incorrectly.